utilities/rate_limiters/write_amp_based_rate_limiter.cc

```diff
@@ -26,6 +26,23 @@ struct WriteAmpBasedRateLimiter::Req {
   bool granted;
 };
 
+namespace {
+constexpr int kSecondsPerTune = 1;
+constexpr int kMillisPerTune = 1000 * kSecondsPerTune;
+constexpr int kMicrosPerTune = 1000 * 1000 * kSecondsPerTune;
+
+// Two reasons for adding padding to baseline limit:
+// 1. compaction cannot fully utilize the IO quota we set.
+// 2. make it faster to digest unexpected burst of pending compaction bytes,
+// generally this will help flatten IO waves.
+// The calculation is based on the empirical value of 16%, with special
+// care for low-band.
+int64_t CalculatePadding(int64_t base) {
+  assert(base > 7684898);
+  return 16 * base / 100 + 25208063285369ll / (base - 7684898);
```

Comment:
the magic number confuses me. What are they about?

Reply:
Check the comment above, they are mainly for smoothly enlarging padding for low-pressure.

Reply:
can you give them some meaning names, I don't know how the values of them are decided?

Reply:
;( but there isn't. I could comment some coordinates for better illustration, but the values themselves are pure mathematics.